src/main/java/tfb/status/view/Results.java

```diff
@@ -117,6 +117,13 @@
    */
   public final @Nullable GitInfo git;
 
+  /**
+   * The test metadata, generated during this run (same as test_metadata.json).
+   * Test metadata was not always included in the results, so this will be
```

Comment:
This currently has the wrong name in TFB and you're planning to rename it, right?  So we don't have an exact date for when it was added yet?

Reply:
This PR fixed the naming, it was merged Jan 24. https://github.com/TechEmpower/FrameworkBenchmarks/pull/5424

Reply:
In that case, can you add [a message like this](https://github.com/TechEmpower/tfb-status/blob/f67ee2908b459adab597cf034503acd315bdf7c3/src/main/java/tfb/status/view/Results.java#L25-L26) to the documentation for this field?